tests/test_mdns_service.js

```diff
@@ -0,0 +1,29 @@
+var dns_sd = require('../lib/dns_sd')
+  , proxyquire =  require('proxyquire')
```

Comment:
No, not globally installed.  It's in devDependencies:

https://github.com/agnat/node_mdns/blob/e38418c22ffc8d7db09e1bc689f617a4f3c6e9a5/package.json#L18